webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/AppConfigAuth.scala

```diff
@@ -633,7 +659,12 @@ class AuthBackendProvidersManager() extends DynamicRudderProviderManager {
   // add default providers
   this.addProvider(defaultAuthBackendsProvider)
   this.addProviderProperties(
-    Map(DefaultAuthBackendProvider.FILE -> AuthBackendProviderProperties(ProviderRoleExtension.WithOverride))
+    Map(
+      DefaultAuthBackendProvider.FILE -> AuthBackendProviderProperties(
+        ProviderRoleExtension.WithOverride,
+        restTokenFeatureSwitch = FeatureSwitch.Enabled
```

Comment:
I don't get why we have an `Enabled` here?

Reply:
All providers must behave the same. 
The only one that must be different is the "super user" provider (the one account that can be configured in properties with `rudder.auth.admin.login`/`rudder.auth.admin.password` : this one must *ALWAYS* be disabled (ie, it can't use API at all).

Reply:
this one will be overridden (the config by map by provider and we always traverse values from `rudder.user.provider`), I think the operation is irrelevant and can be removed here, but I will test that